docker-compose.yml

```diff
@@ -11,6 +11,30 @@ services:
       MYSQL_CONNECTION_STRING: "mysql2://ht_repository:ht_repository@mariadb/ht_repository"
       PUSHGATEWAY: http://pushgateway:9091
 
+  phctl:
+    build: .
+    volumes:
+      - .:/usr/src/app
+      - gem_cache:/gems
+    environment:
+      REDIS_URL: redis://redis/
+      MYSQL_CONNECTION_STRING: "mysql2://ht_repository:ht_repository@mariadb/ht_repository"
+      PUSHGATEWAY: http://pushgateway:9091
+    entrypoint: bundle exec ruby bin/phctl.rb
+
+  processor:
+    build: .
+    restart: always
```

Comment:
Should the phctl instance have restart:always as default? 
Seems potentially harmful.

Reply:
I think this is correct. Sidekiq should manage retrying individual jobs, and if the processor (f/k/a "worker") crashes over and over again Kubernetes will mark it as `CrashLoopBackOff`; `docker-compose` has similar functionality to prevent a restart loop.